tests/test_units.py

```diff
@@ -20,203 +20,221 @@
-
-        assert quantity_store.get_quantity('dimensionless') == unit_registry.dimensionless
+    def test_quantity_translation(self, unit_store):
+        """Tests that unit equality is correctly determined."""
 
         # Units defined in the test CellML <model>:
         for name in TestUnits.test_definitions.keys():
-            assert quantity_store.get_quantity(name) != unit_registry.Quantity(1)
-
-        # Pint built-in units
-        assert quantity_store.is_unit_equal(
-            quantity_store.get_quantity('kilogram'),
-            unit_registry.kilogram
-        )
+            assert unit_store.is_unit_defined(name)
 
         # Custom units defined in CellML example
-        assert quantity_store.is_unit_equal(
-            quantity_store.get_quantity('per_ms'),
-            unit_registry.millisecond**(-1)
+        '''
+        assert
+
+        a = 1 * unit_store.get_quantity('second')
+        b = 1 * unit_store.get_quantity('ms')
+
+        print(a.to_base_units())
+        print(b.to_base_units())
+
+
+
+
+        assert unit_store.is_unit_equal(
+            1 * unit_store.get_quantity('per_ms'),
+            1 / (unit_store.get_quantity('second') / 1000)
         )
 
-        assert quantity_store.is_unit_equal(
-            quantity_store.get_quantity('mM_per_ms'),
+        assert unit_store.is_unit_equal(
+            unit_store.get_quantity('mM_per_ms'),
             (unit_registry.milli_mole / unit_registry.liter) / unit_registry.ms
         )
 
-        assert quantity_store.is_unit_equal(
-            quantity_store.get_quantity('mV_per_usec'),
+        assert unit_store.is_unit_equal(
+            unit_store.get_quantity('mV_per_usec'),
             unit_registry.mV / unit_registry.usec
         )
 
-        assert quantity_store.is_unit_equal(
-            quantity_store.get_quantity('ms_power_prefix'),
+        assert unit_store.is_unit_equal(
+            unit_store.get_quantity('ms_power_prefix'),
             unit_registry.millisecond
         )
 
-        assert quantity_store.is_unit_equal(
-            quantity_store.get_quantity('ms_with_multiplier'),
+        assert unit_store.is_unit_equal(
+            unit_store.get_quantity('ms_with_multiplier'),
             unit_registry.millisecond
         )
+        '''
 
-    def test_conversion_factor(self, quantity_store):
+    def test_conversion_factor(self, unit_store):
         """ Tests Units.get_conversion_factor() function. """
-        ureg = quantity_store.ureg
-        assert quantity_store.get_conversion_factor(quantity=1 * ureg.ms, to_unit=ureg.second) == 0.001
-        assert quantity_store.get_conversion_factor(quantity=1 * ureg.volt, to_unit=ureg.mV) == 1000.0
-
-        assert quantity_store.get_conversion_factor(
-            quantity=1 * quantity_store.get_quantity('milli_mole'),
-            to_unit=quantity_store.get_quantity('mole')
+        assert unit_store.get_conversion_factor(
+            quantity=1 * unit_store.get_quantity('ms'),
```

Comment:
Given the method name is `get_quantity` do we need the `1 * `? Or is the method really `get_unit`?

Should the `to_unit` argument on the next line be changed?

Reply:
`get_quantity` should definitely be `get_unit`. Will check the rest later!